pkg/cloudprovider/providers/oci/load_balancer_spec_test.go

```diff
@@ -566,21 +566,6 @@ func TestNewLBSpecFailure(t *testing.T) {
 			},
 			expectedErrMsg: "error parsing service annotation: service.beta.kubernetes.io/oci-load-balancer-connection-idle-timeout=whoops",
 		},
-		"missing subnet defaults and annotations": {
```

Comment:
Would be worth changing this to a negative test case perhaps. Ensure that a configuration with single subnet1 but empty subnet2 is valid?